### PR TITLE
AZP: proto_enable on gpu

### PR DESCRIPTION
**Pull request by evgeny-leksikov**
## What
proto_enable on gpu (warn)

## Why ?
test coverage

### CODE REVIEW

File: buildlib/pr/tests.yml [hunk 1]
```diff
@@ -30,7 +30,13 @@ jobs:
           source ./buildlib/az-helpers.sh
           az_init_modules
           ./contrib/test_jenkins.sh
-          exit $?
+          rc=$?
+          if [[ $rc != 0 && x'${{ parameters.proto_enable }}' == x'yes' && \
+                x'${{ parameters.ucx_gpu }}' == x'yes' ]]; then
```
Comment (evgeny-leksikov) [inline buildlib/pr/tests.yml:35]: x'${{ parameters.demands }}' == x'ucx_gpu -equals yes'

File: buildlib/pr/main.yml [hunk 2]
```diff
@@ -263,6 +263,12 @@ stages:
         demands: ucx_roce -equals yes
         test_perf: 0
         proto_enable: yes
+    - template: tests.yml
+      parameters:
+        name: gpu_proto_enable
+        demands: ucx_gpu -equals yes
+        test_perf: 0
+        proto_enable: yes
```
Comment (yosefe) [inline buildlib/pr/main.yml:271]: this will double the load on GPU machines and can lead to a longer time to get CI results

Comment (yosefe) [reply buildlib/pr/main.yml:271]: IMO we should keep this PR open to fix the new CI failures

Comment (evgeny-leksikov) [reply buildlib/pr/main.yml:271]: if we won't merge this PR then maybe report failures as error?
I think we could add new variants to tests failed in this PR to cover related code.

Comment (yosefe) [reply buildlib/pr/main.yml:271]: > if we won't merge this PR then maybe report failures as error?

Yes, let's do that

**Comment by artemry-nv** [comment]
Is it still actual?

**Comment by evgeny-leksikov** [comment]
new protocols are enabled by default now, closing

**Pull request closed without merging**